client/my-sites/customer-home/cards/education/earn/payments-features-modal.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { BlankCanvas } from 'calypso/components/blank-canvas';
+import VideosUi from 'calypso/components/videos-ui';
+import ModalFooterBar from 'calypso/components/videos-ui/modal-footer-bar';
+import ModalHeaderBar from 'calypso/components/videos-ui/modal-header-bar';
+import { COURSE_SLUGS } from 'calypso/data/courses';
+import '../blogging-quick-start/style.scss';
+
+const PaymentsFeaturesModal = ( props ) => {
+	const { isVisible = false, onClose = () => {} } = props;
+
+	// Scroll to top on initial load regardless of previous page position
+	useEffect( () => {
+		if ( isVisible ) {
+			window.scrollTo( 0, 0 );
+		}
+	}, [ isVisible ] );
+
+	return (
+		isVisible && (
+			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+			<BlankCanvas className={ 'blogging-quick-start-modal' }>
```

Comment:
Nitpick: This can be simplified given that we don't need to run any JS code
```suggestion
			<BlankCanvas className="blogging-quick-start-modal">
```